app/models/user.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class User < ApplicationRecord
-  devise :registerable, :trackable, :confirmable, :yolo_authenticatable
+  devise :trackable, :yolo_authenticatable
```

Comment:
Never got to ask last time: how does the yolo strategy work?

Reply:
you enter an email address and it just lets you in without a password or magic link